### PR TITLE
tekton: update bugfix-release.sh script for ghcr.io

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

Following the update of the cheatsheet, updating the script to pick up
the changes.

/kind misc
/cc @afrittoli 

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by vdemeester** [comment]
@afrittoli ^^

File: tekton/bugfix-release.sh [hunk 1]
```diff
@@ -41,11 +41,19 @@ echo "> Starting the release pipeline"
 tkn pipeline start pipeline-release \
   --serviceaccount=release-right-meow \
   --param=gitRevision="${TEKTON_RELEASE_GIT_SHA}" \
+  --param=imageRegistry=ghcr.io \
+  --param=imageRegistryRegions="" \
+  --param=imageRegistryPath=tektoncd/pipeline \
+  --param=imageRegistryRegions="" \
```
Comment (afrittoli) [inline tekton/bugfix-release.sh:47]: Duplicated from L44

**Review by afrittoli — COMMENTED** [review]
Just one lit, looks good otherwise
/approve

**Review by afrittoli — COMMENTED** [review]
/lgtm

**Pull request merged**